libs/scully-schematics/src/plugin-ts/index.ts

```diff
@@ -61,8 +57,8 @@ const updateScullyConfig = (options: Schema) => (tree: Tree, context: SchematicC
   if (!scullyJs) {
     context.logger.error(`No scully configuration file found ${scullyConfigFile}`);
   }
-  const newScullyJs = addTypescriptFolder(scullyJs, `pluginDir: './scully/plugins/'`);
+  // const newScullyJs = addTypescriptFolder(scullyJs, `pluginDir: './scully/plugins/'`);
 
-  tree.overwrite(scullyConfigFile, newScullyJs);
-  context.logger.info(`✅️ Update ${scullyConfigFile}`);
+  // tree.overwrite(scullyConfigFile, newScullyJs);
+  // context.logger.info(`✅️ Update ${scullyConfigFile}`);
```

Comment:
YOu need the newScully for replace the file (or add) and the logger for show the message